package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "poetry",
+  "version": "1.0.0",
+  "main": "expo-router/entry",
+  "scheme": "myapp",
+  "web": {
+    "bundler": "metro"
+  },
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@react-navigation/native": "^6.1.9",
+    "@react-navigation/native-stack": "^6.9.17",
+    "@testing-library/react-native": "^12.4.1",
+    "@types/react": "~18.2.14",
+    "@types/react-test-renderer": "^18.0.7",
+    "expo": "~49.0.15",
+    "expo-constants": "~14.4.2",
+    "expo-linking": "~5.0.2",
+    "expo-router": "^2.0.0",
+    "expo-status-bar": "~1.6.0",
+    "jest": "^29.7.0",
+    "jest-expo": "^49.0.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.72.6",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-safe-area-context": "4.6.3",
+    "react-native-screens": "~3.22.0",
+    "react-native-web": "~0.19.6",
+    "ts-jest": "^29.1.1",
+    "typescript": "^5.1.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.0",
+    "@types/jest": "^29.5.10"
+  },
```

Comment:
`jest`, `@testing-library` and `@types` related libs should be in `devDepencencies`